meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-client.sh

```
@@ -0,0 +1,309 @@
+    esac
+done
+
+COMMAND="$1"
+shift
+
+case "$COMMAND" in
+    containers)
+        echo -e "${BOLD}Docker Containers:${NC}\n"
+        response=$(curl -s "${BASE_URL}/containers")
+
+        if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)